src/app/map/components/drawing-edit-overlay/drawing-edit/drawing-edit.component.ts

```diff
@@ -11,12 +12,19 @@ import {PointEditComponent} from './point-edit/point-edit.component';
 import {LineEditComponent} from './line-edit/line-edit.component';
 import {PolygonEditComponent} from './polygon-edit/polygon-edit.component';
 import {TextEditComponent} from './text-edit/text-edit.component';
+import {SymbolEditComponent} from './symbol-edit/symbol-edit.component';
+import CIMSymbol from '@arcgis/core/symbols/CIMSymbol';
+import WebStyleSymbol from '@arcgis/core/symbols/WebStyleSymbol';
+import {scaleCIMSymbolTo, applyCIMSymbolRotation} from '@arcgis/core/symbols/support/cimSymbolUtils.js';
 
+function isGb3SymbolStyle(style: Gb3StyleRepresentation): style is Gb3SymbolStyle {
```

Comment:
If it's used as some sort of utils, I would move it to a utils file; otherwise, it can also be a (private) component method, if it's only used here.